spec/cypress/integration/admin-cip-stories.js

```diff
@@ -0,0 +1,94 @@
+describe("Admin user interaction with Core Induction Programme", () => {
+  beforeEach(() => {
+    cy.login("admin");
+  });
+
+  it("should show a download export button", () => {
+    cy.visit("/core-induction-programme");
+    cy.get("a.govuk-button").contains("Download export").should("exist");
+  });
+
+  it("should allow to edit year title", () => {
+    cy.appFactories([["create", "course_year"]]);
+
+    cy.visitYear();
+    cy.get("h1").should("contain", "Test Course year");
+    cy.get("a.govuk-button").contains("Edit year content").should("exist");
+    cy.get("a.govuk-button").contains("Edit year content").click();
+
+    cy.get("h1").should("contain", "Content change preview");
+    cy.get("input[name='title']").type("New title");
+    cy.get("input").contains("See preview").click();
```

Comment:
I'm uncomfortable using element tag names as selectors in feature-focused tests. This could change to a button and the user wouldn't care, but the test would start failing.

Cypress have some recommendations on their best practice page around selectors: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

Reply:
Eh, maybe you are right.

On the other hand, we are using govuk form builder to create submits, so I imagine it won't change any time soon.

I switched to just using text, and it seems to work.